elastic/kibana
# [Security Solution] Saving is not working if hitting "favorites button" beforehand

## Summary

Issue description and how to reproduce: 
https://github.com/elastic/kibana/issues/85403

Expected result:
https://github.com/elastic/kibana/issues/85403#issuecomment-762733835

----------------------------

This PR also fixes event details not able to be loaded in notes tab after clicking on `>`

![Screenshot 2021-01-21 at 23 01 15 (2)](https://user-images.githubusercontent.com/6295984/105378091-5c696380-5c03-11eb-94db-aaddd02bb59c.png)

I found no data from serverside:
![Screenshot 2021-01-30 at 03 23 38](https://user-images.githubusercontent.com/6295984/106318425-d931b880-6267-11eb-8ce8-9d38ea678f6f.png)

I am able to reproduce it with postman.
<img width="1349" alt="Screenshot 2021-01-30 at 03 23 15" src="https://user-images.githubusercontent.com/6295984/106318435-dc2ca900-6267-11eb-9db5-ce90731aad9a.png">

I logged the error message from server side:
![Screenshot 2021-01-30 at 03 23 07](https://user-images.githubusercontent.com/6295984/106318466-e3ec4d80-6267-11eb-8f76-f3b5512f5035.png)

It fails twice in my 30 trials:
![bsearch-eventdetails2:30failed](https://user-images.githubusercontent.com/6295984/106318482-e9e22e80-6267-11eb-9c20-e42dd7b6dc83.gif)





Step to reproduce:
1. Open any of a timeline with events in it.
2. Leave an event note.
3. Go to notes tab and clicking on `>`
4. The failed to search error occurs

Expected result:
Event details should be loaded properly.

### Checklist
